spec/models/bucket_list_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe BucketList, type: :model do
+  subject { FactoryGirl.build(:bucket_list) }
+  it { should have_many(:users)}
+  # it { should have_many(:destinations)}
+
+  it { should have_valid(:title).when('New Mexico', 'South East Asia with Kam') }
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.